library/ssl_tls13_keys.c

```diff
@@ -455,25 +455,27 @@ int mbedtls_ssl_tls13_derive_early_secrets(
      */
 
     /* Create client_early_traffic_secret */
-    ret = mbedtls_ssl_tls13_derive_secret(hash_alg,
-                                          early_secret, hash_len,
-                                          MBEDTLS_SSL_TLS1_3_LBL_WITH_LEN(c_e_traffic),
-                                          transcript, transcript_len,
-                                          MBEDTLS_SSL_TLS1_3_CONTEXT_HASHED,
-                                          derived->client_early_traffic_secret,
-                                          hash_len);
+    ret = mbedtls_ssl_tls13_derive_secret(
+        hash_alg,
+        early_secret, hash_len,
+        MBEDTLS_SSL_TLS1_3_LBL_WITH_LEN(c_e_traffic),
+        transcript, transcript_len,
+        MBEDTLS_SSL_TLS1_3_CONTEXT_HASHED,
+        derived->client_early_traffic_secret,
+        hash_len);
```

Comment:
It should be indent base on `mbedtls_ssl_tls13_derive_secret` not `ret =` . But I am not very sure if code style script happy with that.
```suggestion
    ret = mbedtls_ssl_tls13_derive_secret(
              hash_alg,
              early_secret, hash_len,
              MBEDTLS_SSL_TLS1_3_LBL_WITH_LEN(c_e_traffic), 
              transcript, transcript_len,
              MBEDTLS_SSL_TLS1_3_CONTEXT_HASHED,
              derived->client_early_traffic_secret,
              hash_len);
```

Reply:
Seems code style is not happy about that. About this topic, I just talked about it with @gilles-peskine-arm , and he preferred to leave it as it was.